packages/azureblobstorage/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@openfn/language-azureblobstorage",
```

Comment:
@josephjclark Azure have other storage services, [see Azure storage data services here]( https://learn.microsoft.com/en-us/azure/storage/common/storage-introduction?toc=/azure/storage/blobs/toc.json&bc=/azure/storage/blobs/breadcrumb/toc.json#azure-storage-data-services), Does this mean we have to create an adaptor for each storage service when they are needed ?

Reply:
Well not necessarily - but I'm wondering if it might be a good pattern to adopt. 

It will mean easier-to-understand adaptors, shorter docs, tighter APIs.

A big consideration is whether you want to use two services in a single job. If it's common to use blob storage and files in the same job/task, then it makes sense to consolidate.

And yes, rather than just an `azure-blobstorage`, it might make sense to have `azure-storage`.

Just thoughts for future adaptors!